src/QMCHamiltonians/ECPComponentBuilder.1.cpp

```diff
@@ -154,7 +154,7 @@ void ECPComponentBuilder::buildLocal(xmlNodePtr cur)
   {
     if (grid_global == 0)
     {
```

Comment:
Please remove {} for a single line. Please update all the places touched.

Reply:
fixed